<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeDockWidget</class>
 <widget class="QDockWidget" name="SizeDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Size</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="styleSheet">
    <string notr="true">#dockWidgetContents {
background-color: #D3DADF;
}

#padBtn, #padBtnL, #phoneBtn, #phoneBtnL, #phonehBtn, #phonehBtnL {
background-image: none;
background-repeat: no-repeat;
border: 0px;
}</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>W:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>H:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="dpiLabel">
       <property name="text">
        <string>DPI:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dpiEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="phoneBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/device/images/phone-portrait.png</normaloff>:/device/images/phone-portrait.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="phonehBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/device/images/phone568h-portrait.png</normaloff>:/device/images/phone568h-portrait.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="padBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/device/images/ipad-portrait.png</normaloff>:/device/images/ipad-portrait.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="phoneBtnL">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/device/images/phone-landscape.png</normaloff>:/device/images/phone-landscape.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="phonehBtnL">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/device/images/phone568h-landscape.png</normaloff>:/device/images/phone568h-landscape.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="padBtnL">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/device/images/ipad-landscape.png</normaloff>:/device/images/ipad-landscape.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
